### PR TITLE
Feature/keep from aws and master

### PR DESCRIPTION
**Pull request by elisehuard**

### CODE REVIEW

**Comment by jasebell** [comment]
lgtm :shipit:

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
...

## Mac OS X users

Mac OS X seems to have an old grep version by default. to fix this, the homebrew version is a solution:

```
brew install grep --with-default-names
```

this will replace grep.  Make sure your PATH has /usr/local/bin preceding /usr/bin or /bin
```
Comment (jasebell) [inline README.md]: where's the grep? ;)

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
...
```
Comment (otfrom) [inline README.md]: git checkout should be git clone IIUC

**Comment by sw1nn** [comment]
How about also adding checking that the e-mail address is a @mastodonc.com one?

Something like: 

```
if [[ ! "${GIT_COMMITTER_EMAIL}" =~ "@mastodonc.com" ]]; then
    echo -e "\nERROR:"
    echo -e "Must use a @mastodonc.com email address:"
    echo -e "Run something like this in the top level of the repo:"
    echo -e "\n\tgit config user.email <name>@mastodonc.com\n"
    exit 1
fi
```

**Comment by elisehuard** [comment]
Not sure?  I know for a fact that not everyone uses a mastodonc address.  Unless we want to make this policy explicitly ...

File: README.md (hunk 2 above)
Comment (elisehuard) [inline README.md]: done, thanks

**Comment by otfrom** [comment]
@elisehuard I think Tommy Hall (and the few others) can tweak this script as needed. wdyt?

**Comment by elisehuard** [comment]
Possibly, however it's a good start for now?

**Comment by otfrom** [comment]
:shipit:

**Pull request merged**